src/providers/twitch/TwitchChannel.cpp

```diff
@@ -944,6 +945,10 @@ void TwitchChannel::createClip()
         return;
     }
 
+    if (QTime().currentTime().msecsTo(this->timeNextClipCreationAllowed_) > 0)
```

Comment:
Can we use [`QTime::operator<`](https://doc.qt.io/qt-5/qtime.html#operator-lt) instead?

Reply:
I assumed calling provided method is better since it exists just for this, but I guess your solution is cleaner.